modules/client-configuration/pages/supported-features-oracle.adoc

```diff
@@ -0,0 +1,66 @@
+[[supported-features-oracle]]
+= Supported Oracle Features
+
+
+This table lists the availability of various features on {oracle} clients.
+
+The icons in this table indicate:
+
+* icon:check[role="green"] the feature is available on both traditional and Salt clients
+* icon:times[role="danger"] the feature is not available
+* icon:question[role="gray"] the feature is under consideration, and may or may not be made available at a later date
+* ``Traditional`` the feature is supported only on traditional clients
+* ``Salt`` the feature is supported only on Salt clients
+
+
+[cols="1,1,1,1", options="header"]
+.Supported Features on {oracle} Operating Systems
```

Comment:
Same about this matrix :-)

Reply:
yep, going to spend some time on all these chapters in the next few weeks. I can't go through this pain every damn release.